src/bin/process_helper/handler.rs

```diff
@@ -1,31 +1,51 @@
 use std::path::PathBuf;
 use std::sync::Arc;
 
+use anyhow::{anyhow, Result};
 use compositor_chromium::cef;
 use compositor_render::{EMBED_SOURCE_FRAMES_MESSAGE, UNEMBED_SOURCE_FRAMES_MESSAGE};
-use log::{error, warn};
-use shared_memory::ShmemConf;
+use log::{debug, error};
 
-use crate::state::{FrameInfo, Source, State};
+use crate::state::{FrameInfo, State};
 
 pub struct RenderProcessHandler {
     state: Arc<State>,
 }
 
 impl cef::RenderProcessHandler for RenderProcessHandler {
+    fn on_context_created(
+        &mut self,
+        _browser: &cef::Browser,
+        _frame: &cef::Frame,
+        context: &cef::V8Context,
+    ) {
+        let mut global = context.global().unwrap();
+        let ctx_entered = context.enter().unwrap();
+
+        context.eval(include_str!("render_frame.js")).unwrap();
```

Comment:
I wanted to write everything in rust but I had some problems with rewriting `render_frame.js` to rust so I decided to simply eval it